bin/all_sky_search/pycbc_calculate_dq

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+""" Ingest and process DQ data
+"""
+import logging, argparse, numpy, h5py, multiprocessing, time, copy
+from six.moves import (range, zip_longest)
```

Comment:
Do we still need `six` now that we moved to Python 3?

Reply:
This has been removed